src/main/scala/com/github/mlangc/slf4zio/api/package.scala

```diff
@@ -227,6 +306,9 @@ package object api {
   def getLogger(name: String): Logger =
     LoggerFactory.getLogger(name)
 
+  def getMarker(name: String): Marker =
+    MarkerFactory.getMarker(name)
```

Comment:
`MarkerFactory.getMarker(name)` is actually an effect, so this method should return `UIO[Marker]` in my opinion.

Reply:
How does this differ from line 306 `LoggerFactory.getLogger(name)`? For `UIO[Marker]` there is wrapper fn: 
`
def makeMarker(name: String): UIO[Marker] =
    UIO(getMarker(name))
`
(Analogous to the logic with creating/making `Logger`)

Reply:
Ok. Will fix